util/simple_mixed_compressor.cc

```diff
@@ -0,0 +1,101 @@
+#include "simple_mixed_compressor.h"
```

Comment:
Since both of these files are new, they should actually use the "Meta Platforms" header. See e.g. advanced_compression.h